RDAPI-17071 contribution steps for edit on netlify cms

andreamussap opened the pull request:
Alex, this is the draft for the contribution using Netlify. Here are some questions that we can discuss on tomorrow's meeting.

- As a user I've created a new topic, and after going back to the main site (doc landing page) the new doc is not there. I'm not sure if should add this to the guidelines, that when creating a new topic it won't show up in the menu??

- On the PR's sent by the user, the Check shows "Axway/cla — Contributor has not signed the Contributor License Agreement.", and the merging is blocked. I don't know if it's blocked because I don't have rights to approve/ merge it or because of the license issue.

- As a user, every time I log in with git to update a page, I also have to click to fork the repo, not only the first time.

- Added information on CLA

alexearnshaw commented on content/en/docs/contribution_guidelines/_index.md:
```diff
@@ -52,7 +62,30 @@ To create an issue just click **Create documentation issue** on the upper right
+{{< /alert >}}
+
+* You will receive an email automatically generated by Netlify CMS with the link for the pull request created in the GitHub for your update.
+* The Content team will receive the same email and will work on it as quickly as possible.
```
Don't think this sentence is needed as it's covered above in the section what to expect. It is good that you tell them about the PR though, so it might be worth saying that we (I would just say project maintainers instead of content team?) might request further info from them on the PR?

andreamussap replied:
I think it's needed because these users may not be aware that there's a GitHub flow behind this process, and even if they are, the sender of the email is Netlify not GitHub so I know that this is a bit of repetition but in this case I think it's better be safe than sorry (in case the email goes to Spam), and this last line is good to close up this procedure.

I think that 'We might contact you...' is not needed though because if we need we'll contact them, if not, no :);  and this is also stated in the 'What to expect...'

andreamussap commented:
@alexearnshaw 
Alex, can you please have a look at modifications before I merge the PR?
- Updated first paragraph to say where the images are.
- Moved contrib guidelines images from \static\Images\docbook\images\contributing to \static\Images
- fback from Alex

alexearnshaw commented on content/en/docs/contribution_guidelines/_index.md:
```diff
@@ -52,7 +63,27 @@ To create an issue just click **Create documentation issue** on the upper right
+* Click **Edit on Netlify CMS** on the upper right corner of the page.
+* Sign in to Netlify CMS using your GitHub account.
+* Click to fork the repository.
+* On the Netlify landing page, locate the page you want to change and click to edit it.
```
On the Netlify landing page => On the Netlify CMS landing page
(just to avoid any confusion between Netlify website and Netlify CMS)

alexearnshaw commented:
@andreamussap Looks good, added 1 teeny comment on diff
